PKHeX/Legality/Structures/ValidEncounterMoves.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace PKHeX.Core
+{
+    public class ValidEncounterMoves
+    {
+        public int EncounterSpecies;
```

Comment:
It probably won't matter too much here, but properties are safer than variables. "public int EncounterSpecies { get; set; }"